org-roam-refile.el

```diff
@@ -30,9 +30,70 @@
 ;; Org-roam refile allows you to refile notes to your nodes.
 ;;
 ;;; Code:
+(eval-when-compile
+  (require 'org-roam-macs)
+  (require 'org-macs))
+
 (defvar org-auto-align-tags)
 (defvar org-loop-over-headlines-in-active-region)
 
+(defun org-roam--file-keyword-get (keyword)
+  "Pull a keyword setting from the top of the file."
```

Comment:
Think it might be better to move the point to the top of file for this operation:

`(org-with-point-at 1 ...)`

Reply:
Not necessary here; the `org-collect-keywords` internals does that already.